tests/vmi_hook_sidecar_test.go

```diff
@@ -88,6 +90,7 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 			var originalConfig v1.KubeVirtConfiguration
 			BeforeEach(func() {
 				originalConfig = *util.GetCurrentKv(virtClient).Spec.Configuration.DeepCopy()
+				vmi.ObjectMeta.Annotations = RenderSidecar(hooksv1alpha1.Version)
```

Comment:
In this PR you introduce a new test. Why do you need to modify the Annotations for the existing tests?

Reply:
Because these annotations were being set in `BeforeEach` at the top `Describe` node. Since I am adding the test within the same node, I modified it. I could have achieved the same without modifying the annotations for existing tests, but this seemed to make it clearer to me while doing it, hence.

Reply:
For me, it seems a bit redundant. The `BeforeEach` from the top `Describe` node could just set the default annotations. While in the context of your test, you overwrite those to smth else. I usually try to avoid touching existing code when introducing new stuff, as it reduces the chance to break smth and also makes it easier to backport such patches.

Reply:
Makes sense. I've reverted.